arrows/pytorch/pytorch_aug_resnet_extractor.py

```diff
@@ -0,0 +1,164 @@
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+
+import torch
+import torch.utils.data as data
+import torch.nn as nn
+
+from torchvision import models, transforms, datasets
+from torch.autograd import Variable
+
+from PIL import Image as pilImage
+
+import math
+import random
+
+import numpy as np
+
+from vital.types import BoundingBox
+
+
+def augment_region( input_image, cx, cy, csize, outsize, rot, tflux=6, sflux=0.3, iflux=0.2 ):
+
+    if 'cv2' not in sys.modules:
+        import cv2
```

Comment:
This imports `cv2` (and therefore allows this function, which references `cv2`) to work only when it hasn't been imported before.  Is this intentional?